### PR TITLE
update version and changelog for release Tengine-2.2.2

### PR DESCRIPTION
**Pull request by wangfakang**

### CODE REVIEW

File: CHANGES.cn [hunk 1]
```diff
@@ -1,3 +1,14 @@
+Tengine 2.2.2 [2018-01-26]
+* Feature: 支持异步openssl，可使用相关硬件(如：QAT)进行SSL的卸载 [mrpre]
```
Comment (taoyuanyuan) [inline CHANGES.cn]: openssl -> OpenSSL

File: CHANGES.cn [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+* Feature: 支持TLS1.3，TLS握手0-RTT [jlijian3] 
+* Feature: upstream配置块里面支持include指令 [wangfakang] 
+* Change：更新并修复测试用列 [wangfakang]
```
Comment (taoyuanyuan) [inline CHANGES.cn]: 用列 -> 用例

Comment (wangfakang) [reply CHANGES.cn]: Done, Thx.

File: CHANGES.cn (hunk 1 above)
Comment (wangfakang) [reply CHANGES.cn]: Done

File: CHANGES.te [hunk 3]
```diff
@@ -1,3 +1,27 @@
...
+
+    *) Feature: support asynchronous SSL/TLS mode. (mrpre)
+
+    *) Feature: support TLS1.3 and 0-RTT data. (jlijian3) 
+
+    *) Feature: enabled "include" inside http upstreams. (wangfakang)
+
+    *) Change: update and fixed the test cases. (wangfakang)
+
+    *) Change: Restore the functionality of the ssl_verify_client_exception
```
[the remark this answers is not in the document]
Comment (wangfakang) [reply CHANGES.te]: Done

File: CHANGES.cn [hunk 4]
```diff
@@ -1,3 +1,14 @@
+Tengine 2.2.2 [2018-01-26]
+* Feature: 支持异步openssl，可使用相关硬件进行SSL的卸载 [mrpre]
+* Feature: 支持TLS1.3，TLS握手0-RTT [jlijian3] 
+* Feature: upstream配置块里面支持include指令 [wangfakang] 
+* Change：更新并修复测试用列 [wangfakang]
+* Change: 恢复ssl_verify_client_exception指令 [jinjiu]  
+* Change: 更新Debian相关软件包版本号 [PeterDaveHello]
+* Change: 更新README文档 [taoyuanyuan]
+* Bugfix: 修复upstream_check模块在reload时可能触发段错误 [yongjianchn]
+* Bugfix: 修复pipe模块在freebsd系统中编译报错问题 [MengqiWu]
```
[the remark this answers is not in the document]
Comment (wangfakang) [reply CHANGES.cn]: Done

File: CHANGES.cn [hunk 5]
```diff
@@ -1,3 +1,14 @@
+Tengine 2.2.2 [2018-01-26]
+* Feature: 支持异步openssl，可使用相关硬件进行SSL的卸载 [mrpre]
+* Feature: 支持TLS1.3，TLS握手0-RTT [jlijian3] 
+* Feature: upstream配置块里面支持include指令 [wangfakang] 
+* Change：更新并修复测试用列 [wangfakang]
+* Change: 恢复ssl_verify_client_exception指令 [jinjiu]  
+* Change: 修复upstream_check模块在reload时可能触发段错误 [yongjianchn]
```
[the remark this answers is not in the document]
Comment (wangfakang) [reply CHANGES.cn]: Done

**Comment by chobits** [comment]
ACK

**Comment by taoyuanyuan** [comment]
ACK

**Pull request merged**